exercise2.js

```diff
@@ -3,41 +3,76 @@
 /************************************************************* */
 // Problem 1: Refactor the for() loop to be a while loop.
 //
-
-for(let i=0; i<10; i++) {
+let i = 0;
+while (i < 11) {
     console.log(" the value of i in the loop is : " + i);
+    i++
 }
 
 
-//your code...
-
-
 
 
 /************************************************************* */
 // Problem 2:
 // multiply the sum of 30 added to two by 20.  Divide the product by 10 raised to the power of 2
 //use javascript to compute the value of the above statement. Each individual operation needs to be a function expression. run all the functions after defining them, and print the answer to the console.
+let sum;
+let mult;
+let div;
 
-//your code...
-
+const all = () => {
+    return console.log(mult / div);
```

Comment:
you're going to want to be careful when using the `var` variable like you are doing here.  It can lead to difficult track down bugs, and code that is hard to reason about, because you can access var variables from other functions.  Always try to keep your variables scoped to the functions they are in, using const or let.